src/utils/spo.ts

```diff
@@ -1485,6 +1488,122 @@ export const spo = {
     await request.post(requestOptions);
   },
 
+  async getFileAsListItemByUrl(webUrl: string, url: string, logger?: Logger, verbose?: boolean): Promise<any> {
+    if (verbose && logger) {
+      logger.logToStderr(`Getting the file properties with url ${url}`);
+    }
+
+    const serverRelativePath = urlUtil.getServerRelativePath(webUrl, url);
+    const requestUrl = `${webUrl}/_api/web/GetFileByServerRelativePath(DecodedUrl=@f)?$expand=ListItemAllFields&@f='${formatting.encodeQueryParameter(serverRelativePath)}'`;
+
+    const requestOptions: CliRequestOptions = {
+      url: requestUrl,
+      headers: {
+        'accept': 'application/json;odata=nometadata'
+      },
+      responseType: 'json'
+    };
+
+    const file = await request.get<FileProperties>(requestOptions);
+
+    return file.ListItemAllFields;
+  },
+
+  /**
+  * Updates a list item with system update
+  * @param webUrl Web url
+  * @param requestUrl The base URL without query parameters, pointing to the specific list where the item resides. This URL should represent the list.
```

Comment:
If this URL is meant to point to the list, it would be clearer to name it `listUrl` or better yet `absoluteListUrl` if it needs to be absolute

Reply:
This comment is marked as resolved without any comment, but the URL is still named `requestUrl`. Why is that?

Reply:
Possible confusion with the other change requests